### PR TITLE
Document the kubernetes CoreDNS release procedure

### PR DESCRIPTION
**Pull request by rajansandeep**
Steps to follow after a CoreDNS release to update the image/manifests for CoreDNS in Kubernetes.

### CODE REVIEW

**Comment by rajansandeep** [comment]
@chrisohaver @fturib @miekg

**Comment by fturib** [comment]
Great check list !

File: kubernetes/README.md
Comment (chrisohaver) [inline kubernetes/README.md]: This step seems out of order with the step above it.

File: kubernetes/README.md
Comment (fturib) [inline kubernetes/README.md:13]: I guess we should say here that we need to assign to a Google person that has permission to push to gcr.io. By default, it is Tim Hockin, SIG-Network.

File: kubernetes/README.md
Comment (chrisohaver) [inline kubernetes/README.md]: I think this should be a separate step (not a sub bullet under 1).

File: kubernetes/README.md
Comment (chrisohaver) [inline kubernetes/README.md]: I think this should be a separate step (not a sub bullet under 1).

File: kubernetes/README.md
Comment (fturib) [inline kubernetes/README.md:25]: get the PR merged by requiring needed reviews and approvals.

File: kubernetes/README.md
Comment (chrisohaver) [inline kubernetes/README.md]: ... and do a `/hold` so the PR does not merge by mistake.

File: kubernetes/README.md
Comment (fturib) [inline kubernetes/README.md:48]: Great idea this checklist.

**Comment by miekg** [comment]
Can we automate it?

**Review by fturib — COMMENTED** [review]
I propose some tuning ...

File: kubernetes/README.md
Comment (chrisohaver) [inline kubernetes/README.md]: Note here that this test will not run until the PR is merged. And that it only runs once per day.   i.e. Need to be sure that the actually included merged PR.

File: kubernetes/README.md
Comment (chrisohaver) [inline kubernetes/README.md]: > If there are any manifest updates from the main [CoreDNS deployment]

Isn't this, by definition, always the case?  Do we need to say this?

File: kubernetes/README.md
Comment (rajansandeep) [reply kubernetes/README.md:48]: Thank you! That was @chrisohaver's suggestion.

File: kubernetes/README.md
Comment (fturib) [inline kubernetes/README.md]: 5k nodes scale test ... (I guess)

**Comment by rajansandeep** [comment]
>Can we automate it?

@miekg I don't think it's possible as most steps require PRs/issues to be manually opened and updating the manifest in different places.

File: kubernetes/README.md
Comment (fturib) [reply kubernetes/README.md]: "Hence ensure that tests run include the merged PR" - needs a rephrasing ? or maybe there is a another word for "run" here ?

**Review by fturib — COMMENTED** [review]
just few other very detailed points ...

**Review by fturib — APPROVED** [review]

**Comment by fturib** [comment]
@miekg : I guess you are the one that can merge here.

**Comment by miekg** [comment]
@fturib that would be odd, is this falls under the coredns org

**Pull request merged**
